lua/nvim-treesitter/utils.lua

```diff
@@ -71,4 +71,12 @@ function M.set_jump()
   vim.cmd "normal! m'"
 end
 
+function M.index_of(tbl, obj)
+  for i, o in ipairs(tbl) do
+    if o == obj then
+      return i
+    end
+  end
+end
+
```

Comment:
Isn't there a `vim.tbl_*`thing for that ?

Reply:
*vim.tbl_add_reverse_lookup()* but it mutates the table.

Reply:
I guess the reason why there is not a helper for that is that it almost always hints for the mistake that you switched keys and values. It will always be inefficient. But if we get usages as an ordered list it should be ok.

Reply:
I think that they should be ordered

Reply:
The order depends on how treesitter captures them in the query. I would assume from top to bottom. IF NOT then we would have to order them ourselves 😢

Reply:
Until now everything was ordered. Should we hope that this stays like that in future?

Reply:
I think that's fair to assume.